joomla/joomla-cms
# [4.0] Missing background image

### Summary of Changes
Make sure that the background image (select-bg-active) can be found


### Testing Instructions
This is as the result of a recent mvoe of the searchtool.scss
Make sure you are on a completely up to date branch and that npm i is also current
Go to any searchtool filter and select anything


### Expected result (after)
Blue background with white dropdown button
![image](https://user-images.githubusercontent.com/1296369/72210857-5ea9c580-34b9-11ea-9a8d-568cb71f3553.png)


### Actual result (before)
Blue background only and 404 on the image
![image](https://user-images.githubusercontent.com/1296369/72210846-4043ca00-34b9-11ea-8ebc-1c043682be4f.png)
